tensorflow/buildspec-2-5-neuron.yml

```diff
@@ -1,46 +0,0 @@
-  account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
-  region: &REGION <set-$REGION-in-environment>
-  framework: &FRAMEWORK tensorflow
-  short_version: &SHORT_VERSION 2.5
-  version: &VERSION 2.5.3
```

Comment:
This points to version 2.5.3, however, [inference/buildpsec-neuron.yml](https://github.com/aws/deep-learning-containers/blob/master/tensorflow/buildspec-neuron.yml) points to version 2.5.2. I am not sure why that is the case.

Irrespective, looks like version 2.5 is the latest one for neuron and hence we might not want to delete `buildspec-2-5-neuron.yml`?